tests/js/spec/views/alerts/create.spec.jsx

```diff
@@ -113,7 +108,6 @@ describe('ProjectAlertsCreate', function () {
   });
 
   afterEach(function () {
-    cleanup();
```

Comment:
How are these cleanups happening now? Does another jest plugin add an afterEach that does the cleanup?

Reply:
this is done automatically by jest. It automatically injects afterEach with cleanups into our testing environment. 

https://testing-library.com/docs/react-testing-library/api/#cleanup